example/src/App.tsx

```diff
@@ -52,10 +55,15 @@ export default function App() {
       </View>
     );
   } else {
+    console.log("CAMERA")
     return (
       <Camera
         device={device}
         isActive={true}
+        preset='hd-3840x2160' // 'high' preset captures the best
```

Comment:
Is this something we should hard-code, what happens if a device doesn't have this preset?

Reply:
preset looks for the closet camera format to the defined preset. so It will always fall back to a format even if it isn't the one thats specified it will move on to the next option.